tests/unit/preprocessor/_weighting/test_weighting_landsea_fraction.py

```diff
@@ -1,6 +1,7 @@
 """Unit tests for :mod:`esmvalcore.preprocessor._weighting`."""
 
 import iris
+import iris.fileformats
```

Comment:
is this only for the new `iris=3.1.0`?

Reply:
This is needed for the tests to work with iris 3.1, see [here](https://scitools-iris.readthedocs.io/en/v3.1.0/whatsnew/3.1.html#minimised-imports).

Reply:
this may need to propagate to other tests and/or other modules we have? Agh, will have to give it a one off proper with iris 3.1 next week, we need to drop `pyke` too

Reply:
All tests are passing now, so i guess it's only needed here

Reply:
This shouldn't go into or affect this PR, but probably we should use `iris.analysis.cartography.DEFAULT_SPHERICAL_EARTH_RADIUS` instead of `iris.fileformats.pp.EARTH_RADIUS` anyway.